reference/src/main/java/com/google/android/fhir/reference/FhirApplication.kt

```diff
@@ -29,7 +29,15 @@ import kotlinx.coroutines.launch
 import org.hl7.fhir.r4.model.ResourceType
 
 class FhirApplication : Application() {
-  init {
+  //     only initiate the FhirEngine when used for the first time, not when the app is created
```

Comment:
Remove the unnecessary spaces. Also capitalize the sentence.

```suggestion
  // Only initiate the FhirEngine when used for the first time, not when the app is created.
```